### PR TITLE
CORTX-27956: Implement Action status update framework

### PR DESCRIPTION
**Pull request by gauri-bhosale**
Added Message types constants of components.

Signed-off-by: gauri.bhosale <gauri.bhosale@seagate.com>

# Problem Statement
- differentiate message according to message type.

# Design
- Declare Message types of components as constants.
- https://seagate-systems.atlassian.net/wiki/spaces/PUB/pages/891781277/Action+Status+Update+LLD#Connection

# Coding
-  [X] Coding conventions are followed and code is consistent

# Review Checklist 
- [X ] PR is self reviewed
- [X] JIRA number/GitHub Issue added to PR
- [X] Jira and state/status is updated and JIRA is updated with PR link
- [ X] Check if the description is clear and explained
- [ ] Dependencies on other component(s)? [Y/N]

# Documentation
  Checklist for Author
- [X] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

**Review by indrajitzagade — COMMENTED** [review]
Please fix the codacy issue

**Comment by indrajitzagade** [comment]
@gauri-bhosale This PR should be against "fault_k8s", please correct.

**Comment by gauri-bhosale** [comment]
> @gauri-bhosale This PR should be against "fault_k8s", please correct.

**Review by indrajitzagade — APPROVED** [review]
Please fix the codacy issue.

File: ha/fault_tolerance/const.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
```
[the remark this answers is not in the document]
Comment (gauri-bhosale) [reply ha/fault_tolerance/const.py]: done.

**Review by mariyappanp — APPROVED** [review]

**Review by Madhura-08 — COMMENTED** [review]
Please fix DCO issue

File: ha/fault_tolerance/const.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+
+import enum
+from ha.util.enum_list import EnumListMeta
+
+
+class HEALTH_EVENT_SOURCES(enum.Enum):
```
Comment (indrajitzagade) [inline ha/fault_tolerance/const.py:21]: @gauri-bhosale Lets add metaclass as we have it in below class:

class HEALTH_EVENT_SOURCES(enum.Enum, metaclass=EnumListMeta):

Comment (gauri-bhosale) [reply ha/fault_tolerance/const.py]: done

**Review by ArchanaLimaye — APPROVED** [review]

**Comment by ArchanaLimaye** [comment]
Please fix DCO and get open comments closed

**Comment by gauri-bhosale** [comment]
DCO issue in this PR. created another PR

**Pull request closed without merging**